src/client/components/formComponent/formComponent.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import RenderFormFields from '../renderFormFields/renderFormFields';
+import './formComponent.css';
+
+const FormComponent = ({
+  fieldsToRender,
+  onInputChange,
+  onFormSubmit,
+  userData,
+}) => (
+  <div>
```

Comment:
Please, add className to div container.

Reply:
Sorry, I don't understand why I need the "className" here, could you explain?

Reply:
why do you need to use div in general here? use or fragment or delete it

Reply:
Removed redundant div